ephios/plugins/baseshiftstructures/templates/baseshiftstructures/participation_card_inline.html

```diff
@@ -53,5 +60,9 @@
     {% else %}
         <i class="far fa-clock"></i>
     {% endif %}
-    {{ participation.participant }}
-</a>
\ No newline at end of file
+    {% if  participation.show_participant_data %}
+        {{ participation.participant }}
+    {% else %}
+        <i class="fas fa-user-secret px-1"></i>
```

Comment:
again not sure if that is the best visualization for your fellow volunteers
could also look a bit weird because the badges are suddenly very short?

Reply:
I agree that the wording and icon choice aren't optimal. Lets discuss this in a call?!